packages/lib/utils/frontMatter.ts

```diff
@@ -227,7 +227,7 @@ export const parse = (note: string): ParsedMeta => {
 	if (metadata.is_todo) {
 		if (isTruthy(md['completed?'])) {
 			// Completed time isn't preserved, so we use a sane choice here
-			metadata['todo_completed'] = metadata['user_updated_time'];
+			metadata['todo_completed'] = metadata['user_updated_time'] ?? Date.now();
```

Comment:
Previously, if `updated_time` wasn't set, `todo_completed` would be set to `undefined`. This resulted in tasks being marked as incomplete, even if `completed?` was set to `yes`.